virtcontainers/sandbox.go

```diff
@@ -978,7 +980,10 @@ func (s *Sandbox) startVM() error {
 }
 
 func (s *Sandbox) addContainer(c *Container) error {
-	s.containers = append(s.containers, c)
+	if s.containers[c.id] != nil {
```

Comment:
Same thing here to check a container entry.

Reply:
Good point. Fixed both. Thanks!